2019Labs/CustomSecurityContent/documentation/lab2_profiles.adoc

```diff
@@ -0,0 +1,403 @@
+image::2-02-terminal.png[Terminal]
++
+. Find out the details in the HTML report.
++
+----
+firefox report.html
+----
++
+** The structure of the HTML report is similar to the HTML guide, but it contains the evaluation results.
+** After clicking on the rule title, you can see the detailed rule results.
+** For example, you can see that the rule Set Interactive Session Timeout failed because on the target system there was not any `TMOUT` entry in `/etc/profile`.
+
+.Details of the rule evaluation displayed in the HTML report.
+image::2-03-report.png[Report]
+
+
+== Creating a new rule from scratch
+
+Let’s say that one of the requirements in your corporate policy is that the users have to install the Hexchat application when their laptops travel outside the company. You want to add a check for that to your new profile.
+
+ComplianceAsCode does not have any rule ready for installing this application yet. That means we need to add a new rule for that.
+
+. Find a group directory that fits best your new rule.
+** The rules are located in `linux_os` directory.
+** Rules in the ComplianceAsCode project are organized into groups, which are represented by directories.
+** It only depends on you to decide which group  the new rule belongs to. You can  browse the directory structure to find out.
+** We plan to add a rule about installing a new application, so the `linux_os/guide/system/software/` directory will be a suitable place.
+. Create a new rule directory in the group directory
+** The name of the directory will be the rule ID. Let’s say that `package_hexchat_installed` could be a suitable ID.
++
+----
+mkdir -p linux_os/guide/system/software/package_hexchat_installed
+----
++
+. Create `rule.yml` in the rule directory
+** A description of the rule is stored. Each rule needs to have it.
+** The `rule.yml` is a simple YAML file.
++
+----
+gedit linux_os/guide/system/software/package_hexchat_installed/rule.yml
+----
++
+** Add the following content to the `rule.yml` file using your editor.
++
+----
+documentation_complete: true
+
+title: Install Hexchat Application
+
+description: As of company policy, the travelling laptops have to have the Hexchat application installed.
+
+rationale: The Hexchat application enables IRC communication with the corporate IT support centre.
+
+severity: medium
+----
+.. `documentation_complete: true` means that your rule will not be ignored
+.. `title` is the rule title, which will be displayed on the command line and in SCAP Workbench.
+.. `description` is a section which purpose is to describe the check.
+.. `rationale` should contain a justification why the rule exists.
+.. `severity` can be either `low`, `medium`, or `high`.
++
+. Add the rule ID to the profile selections
+** As described in the previous section, you need to add the ID of your new rule (`package_hexchat_installed`) to the selections list in your profile (`travel.profile`).
+. Add the package to the list of packages
```

Comment:
I would choose another heading - you can use this opportunity to introduce the topic of templates to a greater extent. But try to avoid talking about the implementation, as that one may change before the summit.